DataDog/datadog-agent
# Migrate traps listener to EventPlatform intake

Use epforwarder for the sending of traps instead of log tailers.


### How to test:

1. Install on mimic and run `datadog-agent status`. You should see traps being forwarded or dropped after a certain volume.
2. Run the agent locally and send traps with the `snmptraps` command. Make sure traps are forwarded and can be seen in the datadog UI.
